recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - automake
     - readline 6.2*
     - gmp >=5.0.1,<7
-    - mpfr 3.*
+    - mpfr 4.*
     - mpfi
     - ntl
```

Comment:
While we are rebuilding stuff, can you add pinnings to some of these packages. I know that the the new ntl version has a different soname, so needs a pinning here

Reply:
Ok. I added a pin for `10.*`.